qdrant-landing/content/documentation/embeddings/snowflake.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Snowflake Models
+weight: 1500
+---
+
+# Snowflake
+
+Qdrant supports working with [Snowflake](https://www.snowflake.com/blog/introducing-snowflake-arctic-embed-snowflakes-state-of-the-art-text-embedding-family-of-models/) text embedding models. You can find all the available models on [HuggingFace](https://huggingface.co/Snowflake).
+
+### Setting up the Qdrant and Snowflake models
+
+```typescript
+import {QdrantClient} from '@qdrant/js-client-rest';
+import { pipeline } from '@xenova/transformers';
+
+const client = new QdrantClient({ url: 'http://localhost:6333' });
+
+const extractor = await pipeline('feature-extraction', 'Snowflake/snowflake-arctic-embed-s');
+
+const texts = [
+    "Qdrant is the best vector search engine!",
+    "Loved by Enterprises and everyone building for low latency, high performance, and scale.",
+]
+```
+
+```python
+from qdrant_client import QdrantClient
+from fastembed import TextEmbedding
+
+qclient = QdrantClient(":memory:")
+embedding_model = TextEmbedding("snowflake/snowflake-arctic-embed-s")
+
+texts = [
+    "Qdrant is the best vector search engine!",
+    "Loved by Enterprises and everyone building for low latency, high performance, and scale.",
+]
+```
```

Comment:
Could you please change the order? Python was usually presented as the first language, so it would be great to keep it consistent.